articles/api/management/v2/user-search.md

```diff
@@ -66,17 +71,16 @@ The user data for `jane` would match, but `jane-doe` would not.
 
 The fields which support `raw` subfield queries are as follow:
 
-* `identities.connection﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`identities.provider﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`identities.user_id ﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`email﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`phone_number﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`family_name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`given_name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`username﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`nickname﻿⁠⁠⁠⁠`
-
+* `identities.connection⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`identities.provider⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`identities.user_id⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`email⁠`
+* ⁠⁠⁠⁠`phone_number⁠⁠`
+* ⁠⁠⁠⁠`family_name⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`given_name⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`username⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`name⁠⁠`
+* ⁠⁠⁠⁠`nickname`
 
 ## Example Queries
 
```

Comment:
The `Cross-field search` might be self explanatory for us but I would add the beginning a sentence for this example (Search across all fields for `john`)